src/pydata_sphinx_theme/assets/styles/components/_search.scss

```diff
@@ -93,29 +93,50 @@
     z-index: $zindex-modal;
     top: 30%;
     left: 50%;
-    transform: translate(-50%, -50%);
+    transform: translate(-50%, -30%);
```

Comment:
this change is just to make room below the modal for the results right?

Reply:
Correct